fixtures/flight/package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "flight",
-  "type": "module",
```

Comment:
ESM mode is borked for all except first load. You have to constantly restart the server otherwise. The problem is that the React dispatcher is always null on subsequent loads. I'm not sure what's going on — some kind of ESM/CJS interop module state duplication, perhaps? Didn't figure it out.